### DISCUSSION TITLE
[@types/nodegit] free() is no more needed

### DISCUSSION THREAD

**Discussion by grinish21** [DefinitelyTyped/DefinitelyTyped#66717] [Issues with a @types package]
All the type definitions for [repository](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodegit/repository.d.ts#L58) , [commit](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodegit/commit.d.ts#L78), signature etc have a free() function which as of nodegit [v0.24.0](https://github.com/nodegit/nodegit/releases/tag/v0.24.0) all free functions have been removed. 

I would be more than happy to make the contribution if this seems like a valid change for the repo. 
Thoughts???

[a comment by an automated account is not shown]

**Comment by tniessen** [comment]
@grinish21 Good catch. PR welcome!

**Reply by grinish21** [reply]
I am on it.
